Pod/Classes/public/error/Error.swift

```diff
@@ -0,0 +1,150 @@
+//
+//  Error.swift
+//  Pods
+//
+//  Created by Jake Young on 6/14/16.
+//
```

Comment:
Please update header block to include Wistia copyright and WistiaKit project name (not Pods)